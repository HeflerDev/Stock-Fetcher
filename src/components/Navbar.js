import { Link } from 'react-router-dom';

const Navbar = () => (
  <nav className="board between">
    <Link to="/about" className="col-6 header-button">
      About the Dev
    </Link>
    <Link to="/" className="col-6 header-button">
      Search
    </Link>
  </nav>
);

export default Navbar;
