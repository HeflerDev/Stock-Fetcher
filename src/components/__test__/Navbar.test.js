import renderer from 'react-test-renderer';
import { BrowserRouter, Switch } from 'react-router-dom';
import Navbar from '../Navbar';

describe('when loading navbar', () => {
  test('it renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Switch>
            <Navbar />
          </Switch>
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
