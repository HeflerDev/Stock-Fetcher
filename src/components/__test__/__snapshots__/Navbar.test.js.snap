// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when loading navbar it renders correctly 1`] = `
<nav
  className="board between"
>
  <a
    className="col-6 header-button"
    href="/about"
    onClick={[Function]}
  >
    About the Dev
  </a>
  <a
    className="col-6 header-button"
    href="/"
    onClick={[Function]}
  >
    Search
  </a>
</nav>
`;
